[for-15.05] FreeSWITCH does not build due to missing libedit

```
checking OPUS_CFLAGS... -I/ssdpool/bulktmp/berni/openwrt/openwrt/staging_dir/target-mips_34kc_uClibc-0.9.33.2/usr/include/opus
checking OPUS_LIBS... -L/ssdpool/bulktmp/berni/openwrt/openwrt/staging_dir/target-mips_34kc_uClibc-0.9.33.2/usr/lib -lopus
checking for soundtouch >= 1.7.1... no
checking for flite >= 2.0.0... no
checking for libmongoc-1.0 >= 1.0.8... no
checking for libmemcached >= 0.31... no
checking for librabbitmq >= 0.5.2... no
checking for tap >= 0.1.0... no
checking for libsmpp34 >= 1.10... no
checking for hiredis >= 0.11.0... no
checking for libedit >= 2.11... no
checking for el_line in -ledit... no
checking histedit.h usability... no
checking histedit.h presence... no
checking for histedit.h... no
configure: error: You need to either install libedit-dev (>= 2.11) or configure with --disable-core-libedit-support
configure: WARNING: cache variable V_Package_freeswitch_mod_cv_postinst contains a newline
configure: WARNING: cache variable V_Package_freeswitch_mod_cv_prerm contains a newline
Makefile:931: recipe for target '/ssdpool/bulktmp/berni/openwrt/openwrt/build_dir/target-mips_34kc_uClibc-0.9.33.2/freeswitch-1.5.final/.configured_nyyyyynnynnnynnyynynnnyyynyyynnnnnynynnnnnnnnnnnynnnnnnnyynnnynynnnnnyynnnnnnnnnnnnnnnnnnnnnnnyyynnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnyynnnnnnnnnnnnnnnnnnnnnnnnnnnnnyyyyyyyyyn' failed
make[2]: *** [/ssdpool/bulktmp/berni/openwrt/openwrt/build_dir/target-mips_34kc_uClibc-0.9.33.2/freeswitch-1.5.final/.configured_nyyyyynnynnnynnyynynnnyyynyyynnnnnynynnnnnnnnnnnynnnnnnnyynnnynynnnnnyynnnnnnnnnnnnnnnnnnnnnnnyyynnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnyynnnnnnnnnnnnnnnnnnnnnnnnnnnnnyyyyyyyyyn] Error 1
make[2]: Leaving directory '/ssdpool/bulktmp/berni/openwrt/openwrt/feeds/telephony/net/freeswitch'
package/Makefile:191: recipe for target 'package/feeds/telephony/freeswitch/compile' failed
make[1]: *** [package/feeds/telephony/freeswitch/compile] Error 2
```

 Probably caused by cc2b959905d2ca724629a6531b4b587d54554779 (only in for-15.05)

--- a/net/freeswitch/Makefile
+++ b/net/freeswitch/Makefile
@@ -573,6 +573,7 @@ CONFIGURE_ARGS+= \
 	--sysconfdir="/etc" \
 	--with-modinstdir="/usr/lib/$(PKG_NAME)" \
 	--with-random="/dev/urandom" \
+	--disable-core-libedit-support \
 	$(call autoconf_bool,CONFIG_FS_WITH_BUILTIN_ZRTP,zrtp) \
 	$(call autoconf_bool,CONFIG_FS_WITH_FHS,fhs) \
 	$(call autoconf_bool,CONFIG_FS_WITH_APR_IPV6,ipv6) \

